src/vs/code/electron-browser/issue/issueReporterMain.ts

```diff
@@ -771,7 +771,8 @@ export class IssueReporter extends Disposable {
 			}
 		}
 
-		const queryStringPrefix = product.reportIssueUrl.indexOf('?') === -1 ? '?' : '&';
+		// {{SQL CARBON EDIT}}
+		const queryStringPrefix = repositoryUrl.indexOf('?') === -1 ? '?' : '&';
```

Comment:
did you get this fix accepted upstream in vscode?

Reply:
no I have t create a PR in their repository